docs/layers/github.md

```diff
@@ -51,7 +51,7 @@ func! myspacevim#before() abort
   let g:gista#client#default_username = 'monkeyxite'
 endf
 ```
-Refer [github dashboard](https://github.com/junegunn/vim-github-dashboard), for security concerns you could create a Personal Access Token, export it as an environment variable and use it as a password.
+We recommend generating a [personal access token](https://github.com/settings/tokens) and storing it securely in an environment variable. [Refer to github dashboard](https://github.com/junegunn/vim-github-dashboard) for more information.
```

Comment:
for users who do not Login. the link is 404

Reply:
great catch! I changed it to point to the public help page instead.